README.md

```diff
@@ -2,3 +2,15 @@ Games
 =====
 
 Games in C and C++.
+
+
+# The games:
+C_Hangman.c Hangman
+
+GuessNumber.c	Casino Guessing Game
+
+TicTacToe.cpp
+
+TicTacToewcolor.cpp
+
+moviegame.cpp (movielist.txt)
```

Comment:
Would be happy to have README telling about the games! Can you be more descriptive here?